source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -455,6 +456,30 @@ message TransformationAccessChain {
 
 }
 
+message TransformationAddBitInstructionSynonym {
+
+  // A transformation that adds synonyms for bit instructions by evaluating each pair of bits.
```

Comment:
Can you give a bit more detail?  A small example of how this would work on e.g. two 4-bit integers would be good.

Reply:
Done.